decidim-elections/app/commands/decidim/votings/create_polling_station_results.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    # A command with all the business logic when creating results for a polling station
+    class CreatePollingStationResults < Rectify::Command
+      # Public: Initializes the command.
+      #
+      # form - A form object with the params.
```

Comment:
The description for the polling_officer is missing. 😊

Reply:
Thanks, I've added this to the next PR that builds on top of this one.

Reply:
Yeah, I saw it - **after** reviewing this one. 😄